tests/src/test/java/alluxio/client/fs/ImpersonationIntegrationTest.java

```diff
@@ -233,7 +236,8 @@ public void impersonationHdfsGroupDenied() throws Exception {
   }
 
   private void checkCreateFile(Subject subject, String expectedUser) throws Exception {
-    FileSystemContext context = FileSystemContext.create(subject, ServerConfiguration.global());
+    FileSystemContext context = FileSystemContext.getOrCreate(ClientContext.create(subject,
```

Comment:
why is this test changed?

Reply:
Theoretically before this branch this would have been doing something similar with the original context cache. I changed them to `getOrCreate` instead to see if it would still pass. I did this one specifically because there was a compilation error when I removed the constructor with the `(Subject, AlluxioConfiguration)` parameters